apps/core/components/searchForm/SearchForm.js

```diff
@@ -129,14 +130,59 @@ class SearchForm extends React.Component<Props> {
     }
   };
 
+  handleParamsUpdate = () => {
+    const {
+      travelFrom,
+      travelTo,
+      tripType,
+      adults,
+      infants,
+      bags,
+      limit,
+      onSubmit,
+    } = this.props;
+    if (travelFrom == null || travelFrom.length === 0) {
+      this.props.setAlertContent({
+        message: 'Please choose an origin place',
+      });
+    } else {
+      const dateFrom = format(this.props.dateFrom, BASIC_ISO_DATE_FORMAT);
+      const dateTo = format(this.props.dateTo, BASIC_ISO_DATE_FORMAT);
+      const returnDateFrom = format(
+        this.props.returnDateFrom,
+        BASIC_ISO_DATE_FORMAT,
+      );
+      const returnDateTo = format(
+        this.props.returnDateTo,
+        BASIC_ISO_DATE_FORMAT,
+      );
+      if (onSubmit != null) {
+        onSubmit({
+          dateFrom,
+          dateTo,
+          returnDateFrom,
+          returnDateTo,
+          tripType,
+          travelFrom,
+          travelTo,
+          sort: 'QUALITY',
```

Comment:
Why you don't use the `sortBy`  from the SearchContext?

Reply:
To be honest, I didnt put much thought here. I just copy pasted required logic from handle SubmitPress.
Thats weird 🧐, will replace in both places. But also will try to refactor to avoid copy pasting

Reply:
Changed